# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-hal/examples/halmark/main.rs
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/wgpu-hal/examples/halmark/main.rs
--- a/third_party/rust/wgpu-hal/examples/halmark/main.rs	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/wgpu-hal/examples/halmark/main.rs	Tue Oct 07 17:00:40 2025 +0000
@@ -54,7 +54,7 @@
 
 impl<A: hal::Api> ExecutionContext<A> {
     unsafe fn wait_and_clear(&mut self, device: &A::Device) {
-        device.wait(&self.fence, self.fence_value, !0).unwrap();
+        device.wait(&self.fence, self.fence_value, None).unwrap();
         self.encoder.reset_all(self.used_cmd_bufs.drain(..));
         for view in self.used_views.drain(..) {
             device.destroy_texture_view(view);
@@ -519,7 +519,7 @@
             queue
                 .submit(&[&init_cmd], &[], (&mut fence, init_fence_value))
                 .unwrap();
-            device.wait(&fence, init_fence_value, !0).unwrap();
+            device.wait(&fence, init_fence_value, None).unwrap();
             device.destroy_buffer(staging_buffer);
             cmd_encoder.reset_all(iter::once(init_cmd));
             fence